app/src/test/java/ch/epfl/sdp/appart/AppUserTest.java

```diff
@@ -19,98 +19,98 @@
 
     @Test
     public void userIsCreatedCorrectly() {
-        AppUser user = new AppUser("1234", "Carlo", "carlo.musso@epfl.ch","1234567890");
+        AppUser user = new AppUser("1234", "Carlo", "carlo.musso@epfl.ch","1234567890", "");
```

Comment:
These should be corrected accordingly to the new constructor 

`public AppUser(String userId, String email) { }`